File: python/linear_search.py
def search(arr, search_Element):
	left = 0
	length = len(arr)
	position = -1
	right = length - 1
	for left in range(0, right, 1):
		# left 
		if (arr[left] == search_Element):
			position = left
			print("Element found in Array at ", position +
				1, " Position with ", left + 1, " Attempt")
			break
		# right
		if (arr[right] == search_Element):
			position = right
			print("Element found in Array at ", position + 1,
				" Position with ", length - right, " Attempt")
			break
		left += 1
		right -= 1
	if (position == -1):
		print("With ", left ,"attempt, the array was not found")
arr = [1, 2, 3, 4, 5, 6, 7, 8, 9]
search_element = 1
search(arr, search_element)

File: c++/linearsearch.cpp
//Algorithm to find the required element in a given array
#include <iostream>

using namespace std;
int elem(int n, int arr[]);

int main()
{
    int n;
    int arr[] = {1, 3, 4, 7, 8};
    cout<<"Enter the element to be found: "<<endl;
    cin>>n;
    int res = elem(n, arr);
    if(res == -1) cout<<"Element is not found"<<endl;
    else cout<<"Element is found at position"<<" "<<n<<endl;
    return 0;
}

int elem(int n, int arr[])
{
    for(int i = 0; i < 5; i++)
    {
        if(arr[i] == n) return i;
    }
    return -1;
}
